routes/api/contacts.js

```diff
@@ -1,25 +1,96 @@
-const express = require('express')
+const express = require("express");
+const router = express.Router();
+const { nanoid } = require("nanoid");
+const {
+  listContacts,
+  getContactById,
+  addContact,
+  removeContact,
+  updateContact,
+} = require("../../models/contacts");
+const { contactSchema } = require("../../models/validation");
 
-const router = express.Router()
+router.get("/", async (req, res, next) => {
+  try {
+    const contacts = await listContacts();
+    res.json(contacts);
```

Comment:
please add status to response. I know it can be `200` by default but its good practice to write status anyway